scripts/west_commands/fetchers/http.py

```diff
@@ -5,6 +5,8 @@
 import requests
 
 from west import log
+import sys
+import os
```

Comment:
Keep Python imports together (sorted)

```suggestion
import os
import requests
import sys

from west import log
```